modules/distr-tracing-tempo-getting-the-tempostack-instance-logs.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="getting-tempo-stack-logs_{context}"]
```

Comment:
```suggestion
[id="distr-tracing-tempo-getting-the-tempostack-instance-logs_{context}"]
```
Per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata

> The anchor ID with `_{context}` must be lowercase and match the module’s file name.

Reply:
Thank you, Ashleigh. Fixed.